src/main/java/com/datalogics/pdf/samples/forms/FillForm.java

```diff
@@ -177,8 +177,6 @@ public static void fillAcroformFdf(final PDFDocument pdfDocument, final URL inpu
         final FDFService fdfService = new FDFService(pdfDocument);
         fdfService.importForm(fdfDocument);
 
-        // Run calculations on the AcroForm...
-        FormFieldService.getAcroFormFieldManager(pdfDocument).runCalculateScripts();
```

Comment:
I think the problem is that the calculation order in the form has the barcodes calculated before the calculatedValue. Calculating everything twice makes it converge, and that's why it worked before removing the manual calculations.

So I can see the mechanism by which the results come out that way. I'm still trying to figure out if it's a problem or not, considering that the spec talks about _all_ calculations being done in the calculation order _every time_ a field value changes. What does that mean for import?

I'm going to go see if I can make Acrobat get this wrong too...because if Acrobat pulls through, I think PDFJT should as well.